data/json/effects.json

```diff
@@ -1822,6 +1822,8 @@
     "max_intensity": 3,
     "rating": "bad",
     "blood_analysis_description": "Hypocalcemia"
+    "base_mods": { "speed_mod": [ -5 ], "str_mod": [ -1 ] },
```

Comment:
```suggestion
    "blood_analysis_description": "Hypocalcemia",
    "base_mods": { "speed_mod": [ -5 ], "str_mod": [ -1 ] },
```

Reply:
actually the formatting was all off , the blood analysis description should be at the end if compared to vanilla iron deficiency jsons

Reply:
consistency is probably a good thing